package.json

```diff
@@ -50,6 +50,7 @@
     "cozy-doctypes": "1.82.2",
     "cozy-harvest-lib": "6.4.0",
     "cozy-scripts": "5.10.0",
+    "cozy-sharing": "3.5.0",
```

Comment:
you need to import the CSS file if you want use it https://github.com/cozy/cozy-libs/tree/master/packages/cozy-sharing#how-to-use-the-lib

Reply:
you also need to add the Provider